swift/test/Ice/operations/Server.swift

```diff
@@ -29,7 +29,7 @@ class Server: TestHelperI {
             key: "TestAdapter.Endpoints", value: getTestEndpoint(num: 0))
         let adapter = try communicator.createObjectAdapter("TestAdapter")
         try adapter.add(
-            servant: MyDerivedClassDisp(MyDerivedClassI(self)), id: Ice.Identity(name: "test"))
+            servant: MyDerivedClassI(self), id: Ice.Identity(name: "test"))
```

Comment:
This test checks you can override Object operations by conforming to protocol Ice.Object, or overriding method(s) from a class that conforms to Ice.Object.

The same logic still works after this update.